Added changelog entry for read_* changes (#12)
crossterm-rs/crossterm-winapi: CHANGELOG.md

# master
- Reimplemented `Console::read_single_input_event` and `Console::read_console_input` to be more efficient, safe, and correct

> - `Console::read_console_input` no longer returns a `u32`; the numbr of events is the length of the returned vector.

Reviewer: ```suggestion
- Make `Console::read_console_input` no return an  `u32`; the number of events is the length of the returned vector.
```

Author: Should this read "not return an"?

Reviewer: yes

> - `Console::read_console_input` no longer returns a `u32`; the numbr of events is the length of the returned vector.

Reviewer: ```suggestion
- Make `Console::read_console_input` return an  `u32`; the number of events is the length of the returned vector.

```

Author: To be clear the change was to remove the `u32` from read_console_input

Reviewer: O wait, my bad again. Facepalm. I was in a hurry.

especially data struct.
- Significantly updated the `input` structs, so that winapi native types are no longer exposed to the library by crossterm_winapi structs.
- Removed PartialOrd from types where it didn't really make sense
- Reimplemented `Console::read_single_input_event` and `Console::read_console_input` to be more efficient, safe, and correct

> - `Console::read_console_input` no longer returns a `u32`; the numbr of events is the length of the returned vector.

Reviewer: ```suggestion
- Make `Console::read_console_input` no to return an  `u32`; the number of events is the length of the returned vector.
```

Final comment haha. The idea is just to append 'Make' in order to make it more consistent with the other readme's.

Author: Another typo, but fixed :)
